index.html

```diff
@@ -2030,6 +2030,111 @@ <h4>Presentations Using Derived Credentials</h4>
+presentation</a> in which the self-asserted <a>verifiable credential</a> is
+included. In each of these cases, the <code>CredentialSubject</code> would
+have an <code>id</code> <a>property</a> that matches the <code>id</code> of
+the <a>subject</a>, just as it is done in <a>verifiable credentials</a> that
+are not self-asserted.
+          </p>
+          <p>
+A <a>verifiable presentation</a> that includes a self-asserted
+<a>verifiable credential</a> that is only secured using the same mechanism as
+the <a>verifiable presentation</a> MUST include a <code>holder</code>
+<a>property</a>.
+          </p>
+          <p>
+All of the normative requirements defined for <a>verifiable credentials</a>
+apply to self-asserted <a>verifiable credentials</a>. Some additional
+considerations follow:
+          </p>
+
+        <dl>
+          <dt><var>type</var></dt>
+          <dd>
+It is RECOMMENDED that <code>SelfAssertedCredential</code> be included as an
+additional <code>type</code> value. For example,<br>
+<code>"type": ["VerifiableCredential", "SelfAssertedCredential", "ExampleTypeCredential"]</code>
+          </dd>
+          <dt><var>issuer</var></dt>
+          <dd>
+For self-asserted <a>verifiable credentials</a> that are secured using the same
+mechanism as the <a>verifiable presentation</a>, the value of the
+<code>issuer</code> <a>property</a> MUST be identical to the <code>holder</code>
+<a>property</a> of the <a>verifiable presentation</a>.
+          </dd>
+        </dl>
+        <p>
+The example below shows a <a>verifiable presentation</a> that embeds a
+self-asserted <a>verifiable credential</a> that is secured using the same
+mechanism as the <a>verifiable presentation</a>.
+        </p>
+
+        <pre class="example nohighlight" title="A secured verifiable presentation relying on an embedded proof, including a self-asserted verifiable credential">
+{
+  "@context": [
+    "https://www.w3.org/ns/credentials/v2",
+    "https://www.w3.org/ns/credentials/examples/v2"
+  ],
+  "type": ["VerifiablePresentation", "ExamplePresentation"],
+  "holder": "did:example:12345678",
+  "verifiableCredential": [{
+    "@context": "https://www.w3.org/ns/credentials/v2",
+    "type": ["VerifiableCredential", "SelfAssertedCredential", "ExampleFoodPreferenceCredential"],
+    "issuer": "did:example:12345678",
+    "credentialSubject": {
+      "favoriteCheese": "Gouda"
+    },
+    { <span class="comment">...</span> }
+  }],
+  "proof": [{ <span class="comment">...</span> }]</span>
+}
+        </pre>
+                <p>
+The example below shows a <a>verifiable presentation</a> that embeds a
+self-asserted <a>verifiable credential</a> that holds <a>claims</a> about the
+<a>verifiable presentation</a>. It is secured using the same mechanism as the
+<a>verifiable presentation</a>.
+        </p>
+
+        <pre class="example nohighlight" title="Verifiable presentation with a self-asserted verifiable credential about the verifiable presentation">
```

Comment:
```suggestion
        <pre class="example nohighlight" title="A secured verifiable presentation, relying on an embedded proof including a self-asserted verifiable credential about the verifiable presentation">
```

Reply:
for consistency with https://github.com/w3c/vc-data-model/pull/1186/files#r1264100438

Reply:
@msporny as an aside, I suspect your PR to fix `credential` vs `verifiable credential` will be some version of these changes... over and over again... until you go mad.... : )

Basically replacing `credential` with `unsecured verifiable credential` or `secured verifiable credential` depending on the context... looking forward to that one : )

Maybe we can be clever and do that once in terms, and not have to be so verbose everywhere....

Reply:
> until you go mad....

That happened long ago... during v1.0...

I'll see if we can be clever wrt. terminology, as you mention above... but at this point, I'm leaning toward NOT making the change (as it sounds like it's going to be a lot of trouble, and result in a confusing outcome for readers).

Reply:
I don't see how we can avoid making fixing it, given what it currently normatively states:

- https://w3c.github.io/vc-data-model/#securing-verifiable-credentials

It makes the interpretation of `<a>verifiable credential<a>` and `VerifiableCredential` inconsistent.

The reader learns that embedded or external proof is required for `<a>verifiable credential<a>`.... and optional for `VerifiableCredential`... Seems like we will have to fix that... The same applies to presentations... This PR is digging the whole deeper.... but sometimes a tunnel is necessary to escape.